examples/pygi_mpl_numpy/README.rst

```diff
@@ -66,3 +66,19 @@ The include section requires the Python packages numpy, matplotlib (which are ta
         dateutil
         pyparsing
 
+Compiling
```

Comment:
I was also wondering what the difference was, but I should probably take a few C courses first to fully grasp the concept.